tests/fxl-orientation_default/OPS/nav.xhtml

```diff
@@ -0,0 +1,15 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
```

Comment:
Nit: can you indent all the nav files appropriately (2 spaces for each nested element)? That's done in the templates now but these are old copies.

If you don't do this, that's OK, I'll do it sometime later, since this is just to satisfy my need for consistency, not actually important.